src/React/Basic.purs

```diff
@@ -124,3 +171,21 @@ foreign import createElement_ :: forall props. Fn2 (ReactComponent { | props })
 foreign import createElementKeyed_ :: forall props. Fn2 (ReactComponent { | props }) { key :: String | props } JSX
 
 foreign import fragmentKeyed_ :: Fn2 String (Array JSX) JSX
+
+-- | Deprecated -- prefer `component`
```

Comment:
Let's use `Warn` for this?

Reply:
It might be better to just remove it, if updating requires changes anyway